meerk40t/balormk/device.py

```diff
@@ -450,29 +450,36 @@ def __init__(self, kernel, path, *args, **kwargs):
 
         self.register(
             "format/op cut",
-            "{enabled}{pass}{element_type} {speed}mm/s @{power} {frequency}kHz",
+            "{danger}{defop}{enabled}{pass}{element_type} {speed}mm/s @{power} {frequency}kHz {colcode} {opstop}",
         )
         self.register(
             "format/op engrave",
-            "{enabled}{pass}{element_type} {speed}mm/s @{power} {frequency}kHz",
+            "{danger}{defop}{enabled}{pass}{element_type} {speed}mm/s @{power} {frequency}kHz {colcode} {opstop}",
         )
         self.register(
             "format/op hatch",
-            "{enabled}{penpass}{pass}{element_type} {speed}mm/s @{power} {frequency}kHz",
+            "{danger}{defop}{enabled}{penpass}{pass}{element_type} {speed}mm/s @{power} {frequency}kHz {colcode} {opstop}",
         )
         self.register(
             "format/op raster",
-            "{enabled}{pass}{element_type}{direction}{speed}mm/s @{power} {frequency}kHz",
+            "{danger}{defop}{enabled}{pass}{element_type}{direction}{speed}mm/s @{power} {frequency}kHz {colcode} {opstop}",
         )
         self.register(
             "format/op image",
-            "{enabled}{penvalue}{pass}{element_type}{direction}{speed}mm/s @{power} {frequency}kHz",
+            "{danger}{defop}{enabled}{penvalue}{pass}{element_type}{direction}{speed}mm/s @{power} {frequency}kHz {colcode} {opstop}",
         )
         self.register(
             "format/op dots",
-            "{enabled}{pass}{element_type} {dwell_time}ms dwell {frequency}kHz",
+            "{danger}{defop}{enabled}{pass}{element_type} {dwell_time}ms dwell {frequency}kHz {colcode} {opstop}",
         )
         self.register("format/util console", "{enabled}{command}")
+        # Define maxspeed min, need to be adjusted for balor
+        kernel.register("dangerlevel/op cut", (50, 100))
```

Comment:
For balor, this isn't dangerous at all, at 5000mm/s without issue.

Reply:
Will look into it